pkg/usrlocalsharelima/usrlocalsharelima.go

```diff
@@ -50,14 +51,14 @@ func Dir() (string, error) {
 		filepath.Join(selfDirDir, "share/lima/lima-guestagent."+ostype+"-"+arch),
 		// TODO: support custom path
 	}
-	if logrus.GetLevel() == logrus.DebugLevel {
+	if debugutil.Debug {
```

Comment:
Do we need a new package for keeping one flag?

If we do, I would change the names so we can use:

```go
if debug.Enabled {
    ...
}
```

Reply:
Yes, as it is referred from several pkgs.

I avoided "debug" because it conflicts with variable names